<!-- -->
<launch>
<!--set basic transforms neccessary to get the node running-->


<!--read waypoints and publish next waypoint-->
	<node pkg="goal_manager" type="robot_goal_manager" name="robot_goal_mangager" output="screen">
		<param name="next_point" value="/point"/>
		<param name="next_pose" value="/pose"/>
		<param name="waypoint_list" value="$(find goal_manager)/waypoints/headland_left.txt"/>
		<param name="path_publisher" value="/frame_path"/>
		<param name="wait_for_topic" value="true"/>
		<param name="wait_topic_name" value="/movement_finished"/>

		<param name="rabbit_frame" value="goal_point"/>
		<param name="robot_frame" value="base_link"/>
		<param name="fixed_frame" value="odom"/>
			
		<param name="relative_goals" value="true"/>
		<param name="allow_break_after_seconds" value="false"/>
		<param name="break_after_seconds" value="10"/>
		
		
	</node>
</launch>
